test/test_install_cmdstan.py

```diff
@@ -37,6 +37,8 @@ def test_retrieve_version(self):
         with self.assertRaises(ValueError):
             retrieve_version('')
 
+    from unittest import mock
```

Comment:
If we're adding this import we can change line 42 to just `@mock...`, but I'd just as soon not import this directly

Reply:
it was @mock before but pylint complained.

Reply:
Odd. We definitely shouldn't need the @unittest. if we're importing mock directly, so we can either use @unittest.mock without the import, or @mock with the import

Reply:
These poor github users must get pinged a lot.

Reply:
fixed the @ calls :)

Reply:
```suggestion

```

This import is not necessary for me to pass pylint locally (in fact it's highlighted as unused)